mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/domain/Loan.java

```diff
@@ -1594,7 +1594,7 @@ private void updateDisbursementDetails(final JsonCommand command, final Map<Stri
                             && StringUtils.isNotBlank((jsonObject.get(LoanApiConstants.disbursementIdParameterName).getAsString()))) {
                         id = jsonObject.getAsJsonPrimitive(LoanApiConstants.disbursementIdParameterName).getAsLong();
                     }
-                    LoanDisbursementDetails disbursementDetails = new LoanDisbursementDetails(expectedDisbursementDate, null, principal);
+                    LoanDisbursementDetails disbursementDetails = new LoanDisbursementDetails(expectedDisbursementDate, actualDisbursementDate, principal);
```

Comment:
declare actual disbursedate  locally in method don't refer to loan's actualdisbursementdate